### PR TITLE
[FIX] eyewear_shop: add missing point_of_sale

### PR DESCRIPTION
**Pull request by dhrs-odoo**
Add dependency
Add knowledge section
Add PoS config (data and demo)

Task-3964607

### CODE REVIEW

File: eyewear_shop/data/knowledge_article.xml [hunk 1]
```diff
@@ -115,6 +116,15 @@
                     <li>Select products and proceed to checkout.</li>
                 </ul>
                 <blockquote data-o-mail-quote-node="1" data-o-mail-quote="1">Use discount code "FreedomFest" before checkout to get a discount on your order<br/></blockquote>
+                <h2 data-o-mail-quote-node="1" data-o-mail-quote="1">Flow 5: POS Shopping</h2>
```
Comment (vava-odoo) [inline eyewear_shop/data/knowledge_article.xml:119]: what is those attribute for?

File: eyewear_shop/demo/pos_session.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+        <record id="pos_session_1" model="pos.session">
```
Comment (vava-odoo) [inline eyewear_shop/demo/pos_session.xml:3]: indent :eyes:

File: eyewear_shop/demo/stock_warehouse.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="stock_warehouse_3" model="stock.warehouse">
```
Comment (vava-odoo) [inline eyewear_shop/demo/stock_warehouse.xml:3]: I don't get why you need to define warehouses

File: eyewear_shop/data/pos_config.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        <field name="iface_customer_facing_display_local" eval="True"/>
+        <field name="iface_print_auto" eval="True"/>
+        <field name="iface_customer_facing_display" eval="True"/>
+        <field name="ship_later" eval="True"/>
+        <field name="limit_categories">True</field>
+        <field name="iface_available_categ_ids" eval="[(6, 0, [
+            ref('pos_category_1'),
+            ref('pos_category_2'),
+            ref('pos_category_3'),
+        ])]"/>
```
Comment (vava-odoo) [inline eyewear_shop/data/pos_config.xml:15]: Wasn't it only necessary if we wanted to split categories between different pos.config? It should work without that
```suggestion
```

File: eyewear_shop/__manifest__.py [hunk 5]
```diff
@@ -11,6 +11,7 @@
         'knowledge',
         'payment_demo',
         'purchase_stock',
+        'pos_sale',
```
Comment (vava-odoo) [inline eyewear_shop/__manifest__.py:14]:
```suggestion
        'point_of_sale',
```

**Review by vava-odoo — COMMENTED** [review]
Nice work, thanks! Just a few comments.
Also, could you squash commits into a "_[FIX] eyewear_shop: add missing point_of_sale_" commit?
Cheers

**Comment by dhrs-odoo** [comment]
Hello @vava-odoo 
I have incorporated the suggested changes 
Please have a look at it once you are free 
Thanks , Have a great day ahead :innocent:

**Review by vava-odoo — APPROVED** [review]

**Pull request merged**